### PR TITLE
W3HW6 tests for dataset

### PR DESCRIPTION
**Pull request by yevhen-k**
Hi @truskovskiyk 

I'm not quite sure how the PR should be prepared, so I have 2 options:
- for running locally with `pytest test_dataset.py`
- for running with `great_expectations checkpoint run imdb-checkpoint-1`

### CODE REVIEW

File: week-3-tests/test_dataset.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+DATASET_NAME = "imdb.csv"
+if not os.path.exists(DATASET_DIR):
+    os.mkdir(DATASET_DIR)
+if not os.path.exists(DATASET_DIR+DATASET_NAME):    
+    dataset = get_dataset()
+    df = pd.DataFrame(dataset["train"])
+    df.to_csv(DATASET_DIR+DATASET_NAME, index=False)
+
+df = pd.read_csv(DATASET_DIR+DATASET_NAME)
+df = ge.from_pandas(df)
```
Comment (truskovskiyk) [inline week-3-tests/test_dataset.py]: please use [fixtures](https://docs.pytest.org/en/6.2.x/fixture.html) for this

**Review by truskovskiyk — COMMENTED** [review]
@yevhen-k looks great, please run this with CI

File: week-3-tests/test_dataset.py (hunk 1 above)
Comment (yevhen-k) [reply week-3-tests/test_dataset.py]: done

**Comment by yevhen-k** [comment]
@truskovskiyk tests are green, lease check it

**Review by truskovskiyk — APPROVED** [review]
@yevhen-k amazing!

**Pull request merged**